src/main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.jsx";
-import "./../build/output.css";
+import "../dist/output.css";
```

Comment:
This is never to be done in an application properly configured for Tailwind.  Never.